src/server/biosecurity-map/upload-progress/__snapshots__/index.test.js.snap

```diff
@@ -19,3 +19,5 @@ exports[`#UploadPlan #UploadProgressPage should render expected response and con
   </div>
         "
 `;
+
+exports[`#UploadPlan valid upload should render the upload progress page 1`] = `""`;
```

Comment:
is this correct?

Reply:
this is a snapshot and is 100% generated so i guess so